packages/main/test/specs/ColorPalette.spec.js

```diff
@@ -119,4 +119,27 @@ describe("ColorPalette interactions", () => {
 		assert.strictEqual(await colorPaletteRecentColorsWrapperEntries[3].getProperty("value"), "darkblue");
 		assert.strictEqual(await colorPaletteRecentColorsWrapperEntries[4].getProperty("value"), "pink");
 	});
+
+	it("Tests if only one item is selected at a time in the color palette", async () => {
+		const colorPalette = await browser.$("#cp1SelectedTest");
+		const colorPaletteEntries = await colorPalette.$$("ui5-color-palette-item");
+
+		await colorPaletteEntries[0].click();
+		await colorPaletteEntries[1].click();
+
+
+		assert.strictEqual(await colorPaletteEntries[0].getAttribute("selected"), null, "The first item is not selected");
+		assert.strictEqual(await colorPaletteEntries[1].getAttribute("selected"), "true", "The second item is selected");
+	});
+	
+	it("Tests if clicking on selected item, does not deselect it", async () => {
+		await browser.url(`test/pages/ColorPalette.html`);
```

Comment:
Do we need the page reload? Can we test on another component or base it on previous test?